### ISSUE TITLE
[BUG] [WCAG]/personal-data - Disclaimer text contrast is WCAG non-compliant

### ISSUE THREAD

**Issue by j-isenberg** [reach4help/reach4help#608]
**Describe the bug**
The colour of the disclaimer text "We need this data to ensure the help reaches the correct destination. For any concerns, please read our [...]" does not meet the contrast ratio to be WCAG compliant. 

Currently, the contrast ratio is 1.19:1 WCAG (2.1) requires a contrast ratio of at least  4.5:1 for text. 

URL:  https://reach4help-staging-web-client.firebaseapp.com/personal-data

**To Reproduce**
1. Reach the /personal-data page. 
2. Scroll below the user input fields.  
3. Notice that the text in question is too light in colour to read. 

**Expected behavior**
Text colour is dark enough to be read without screen adjustment and complies with WCAG 2.1 text contrast guidelines. 

**Screenshots**
Text in question: 
![2020-05-29_contrast non-compliant](https://user-images.githubusercontent.com/66037391/83302552-90ea6400-a1c9-11ea-959b-d9dc109769c5.png)

Contrast check: 
![2020-05-29_contrast checker non-compliant](https://user-images.githubusercontent.com/66037391/83302859-09512500-a1ca-11ea-8f52-6606a9f1acbf.png)

**Desktop:**
Windows 10 - Chrome - Incognito mode.

**Additional context**
WCAG quick reference: https://www.w3.org/WAI/WCAG21/quickref/
Contrast checker: https://contrastchecker.com/
Free WCAG compliance tool: https://www.deque.com/axe/

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/608)

*This information is updated automatically. To modify it, please use ZenHub.*

**Comment by schowdhury8** [comment]
Fixed and deployed to https://app.reach4help.org/
